CHANGELOG.md

```diff
@@ -3,13 +3,43 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 4.0.1
+
+_Sep 29, 2021_
+
+This version includes bugfixes and features already released in v5.x.
+
+Big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- 👁 Allow to disable virtualization with the `disableVirtualization` prop (#2728) @m4theushw
```

Comment:
Do we need this feature in v4?

Reply:
I would personally encourage we make this one the last of the v4 release line. Unless there is a major issue: Chrome regression, Node.js regression, webpack regression, etc. Basically, use the policy we had for Core v4 when we were working on Core v5. There are 1-2 years of inertia with open source. v4 keep growing during all the time we were working on v5, while we were almost not making any changes.

Reply:
If MUI X v5 + MUI Core v4 is viable (= if users do not find major issues that we would have missed, like weird style collision) I am 100% in favor of this.

Reply:
> Do we need this feature in v4?

An user asked for it and it was simple to cherry-pick: https://github.com/mui-org/material-ui-x/issues/1902#issuecomment-912017532